### PR TITLE
Data Tab UI Implementation (pt 1)

### PR DESCRIPTION
**Pull request by vaidotasp**
Part of #782
In Scope:
- Prev, Next button styling and functionality
- Layer selector styling and functionality
- Print Report button on the data panel

Out of Scope (and documented in #782)
- "X" button removes attribute from the map and data tab on click
- Active feature on map is highlighted and upon clicking NEXT/PREV highlighting changes to the next feature
- Table styling
- Language awareness

### CODE REVIEW

File: src/css/leftpanel.scss [hunk 1]
```diff
@@ -3,12 +3,12 @@
 .left-panel {
   z-index: 1;
   position: absolute;
-  top: 200px;
+  top: 140px;
```
Comment (SwampGuzzler) [inline src/css/leftpanel.scss:6]: `Change Request` ALERT!! No more `bottom` and we want `height` back in there somehow

File: src/js/components/leftPanel/dataPanel/DataTabFooter.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+const DataTabFooter = (): JSX.Element => {
+  return (
+    <div className="data-tabview-footer">
+      <button onClick={(): void => console.log('Print Report!')}>
+        Print Report{' '}
```
Comment (SwampGuzzler) [inline src/js/components/leftPanel/dataPanel/DataTabFooter.tsx:7]: Idk if we still need the `{' '}` now that our `.data-tabview-container .data-tabview-footer button img` has the proper `margin-left`

File: src/js/components/leftPanel/dataPanel/DataTabFooter.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        <img
+          src="https://my.gfw-mapbuilder.org/img/print-icon.svg"
```
Comment (SwampGuzzler) [inline src/js/components/leftPanel/dataPanel/DataTabFooter.tsx:9]: @vaidotasp Let's eventually steal this `svg` and get it local in our code

File: src/js/components/leftPanel/dataPanel/DataTabView.tsx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        <LayerAttributesElement activeLayerInfo={activeLayerInfo} />
+        <DataTabFooter />
+      </div>
+    );
+  };
+
+  const DataTabViewContent = () => {
+    const tabViewIsVisible = tabViewVisible && activeTab === props.label;
+    if (!tabViewIsVisible) {
+      return null;
```
Comment (SwampGuzzler) [inline src/js/components/leftPanel/dataPanel/DataTabView.tsx]: @vaidotasp Thoughts on whether this Component should be _ripped_ off of the DOM (and more importantly, have its local state of `3 of 14` or whatever be reset) when we have a different tab active (or no tab active), then come back to `1 of 14` every time that we switch tabs? This is Not a change request, let's look at this further down the line.

Comment (SwampGuzzler) [reply src/js/components/leftPanel/dataPanel/DataTabView.tsx]: Ooh let's also check this out again Once We Get The Highlights On The Map For The Selected Feature!

**Review by SwampGuzzler — CHANGES_REQUESTED** [review]
https://github.com/wri/gfw-mapbuilder/pull/805#discussion_r381567388 Style Changes and we are good!

**Comment by vaidotasp** [comment]
yo @SwampGuzzler I think this is a better approach to control the height of the various tab views and their visibility https://github.com/wri/gfw-mapbuilder/pull/805/commits/7b971f83462e7153b073becd671ddeebb6613f57

**Review by SwampGuzzler — APPROVED** [review]
Looks good, as long as we remember to add the `tabview-container` class to all of our tabs Approved with `emphasis!`

**Pull request merged**